coach/src/concept-coach/index.cjsx

```diff
@@ -101,7 +101,7 @@ class ConceptCoachAPI extends EventEmitter2
     @removeAllListeners()
 
   remove: ->
```

Comment:
yeah, I hope so 🙏  I'm not entirely sure where all the coaches stuff gets called from, since this is called into from web view